<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginWin</class>
 <widget class="QWidget" name="LoginWin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>330</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>LoginWin</string>
  </property>
  <widget class="QLabel" name="avatar">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>112</y>
     <width>83</width>
     <height>83</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/pixmap/image/uklogo.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="passwd">
   <property name="geometry">
    <rect>
     <x>114</x>
     <y>155</y>
     <width>207</width>
     <height>34</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/pixmap/image/pw.png);</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="passwdremember">
   <property name="geometry">
    <rect>
     <x>114</x>
     <y>190</y>
     <width>91</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>记住密码</string>
   </property>
  </widget>
  <widget class="QPushButton" name="loginButton">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>115</y>
     <width>90</width>
     <height>74</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Ubuntu Mono</family>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>登录</string>
   </property>
   <property name="iconSize">
    <size>
     <width>16</width>
     <height>16</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="btn_min">
   <property name="geometry">
    <rect>
     <x>388</x>
     <y>10</y>
     <width>26</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/pixmap/image/miniButton.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="btn_close">
   <property name="geometry">
    <rect>
     <x>413</x>
     <y>10</y>
     <width>26</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/pixmap/image/closeButton.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QComboBox" name="username">
   <property name="geometry">
    <rect>
     <x>114</x>
     <y>115</y>
     <width>207</width>
     <height>34</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="editable">
    <bool>true</bool>
   </property>
   <property name="frame">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QWidget" name="bg" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>450</width>
     <height>330</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/pixmap/image/login_bg.png);</string>
   </property>
  </widget>
  <widget class="QLabel" name="btn_setting">
   <property name="geometry">
    <rect>
     <x>363</x>
     <y>10</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/pixmap/image/settingButton.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="btn_register">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>190</y>
     <width>91</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>注册帐号</string>
   </property>
  </widget>
  <widget class="QLabel" name="lotus">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>69</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; color:#ffffff;&quot;&gt;Lotus&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="aboutButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/pixmap/image/aboutButton.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>bg</zorder>
  <zorder>passwdremember</zorder>
  <zorder>avatar</zorder>
  <zorder>btn_close</zorder>
  <zorder>passwd</zorder>
  <zorder>loginButton</zorder>
  <zorder>btn_min</zorder>
  <zorder>username</zorder>
  <zorder>btn_setting</zorder>
  <zorder>btn_register</zorder>
  <zorder>lotus</zorder>
  <zorder>aboutButton</zorder>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>username</tabstop>
  <tabstop>passwd</tabstop>
  <tabstop>passwdremember</tabstop>
  <tabstop>loginButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
